crates/jstzd/tests/utils.rs

```diff
@@ -8,14 +14,14 @@ use octez::r#async::{
     protocol::Protocol,
     rollup::{OctezRollupConfigBuilder, RollupDataDir},
 };
-use regex::Regex;
 use std::path::{Path, PathBuf};
 use tezos_crypto_rs::hash::{BlockHash, OperationHash, SmartRollupHash};
 use tokio::io::AsyncReadExt;
 
 pub const ACTIVATOR_SECRET_KEY: &str =
     "unencrypted:edsk31vznjHSSpGExDMHYASz45VZqXN4DPxvsa4hAyY8dHM28cZzp6";
-pub const ROLLUP_ADDRESS: &str = "sr1PuFMgaRUN12rKQ3J2ae5psNtwCxPNmGNK";
```

Comment:
Does the address need to change?

Reply:
oh forgot to undo this, this address was orignally used in the toy rollup but i changed the toy rollup instead so i can undo this

Reply:
yes, i've changed the toy rollup address in sandbox-params.json due to the conflict w JSTZ_ROLLUP_ADDRESS